src/main/java/at/hannibal2/skyhanni/config/features/combat/broodmother/BroodmotherConfig.java

```diff
@@ -0,0 +1,68 @@
+package at.hannibal2.skyhanni.config.features.combat.broodmother;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.core.config.Position;
+import at.hannibal2.skyhanni.features.combat.BroodmotherFeatures.StageEntry;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.Accordion;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorDraggableList;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigLink;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class BroodmotherConfig {
+
+    @Expose
+    @ConfigOption(name = "Countdown", desc = "Display a countdown until the Broodmother will spawn.\n" +
+        "§cCountdown will not show unless the time until spawn has been established, and may be off by a few seconds.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean countdown = true;
+
+    @Expose
+    @ConfigOption(name = "Spawn Alert", desc = "Send a chat message, title and sound when the Broodmother spawns.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean spawnAlert = false;
+
+    @Expose
+    @ConfigOption(name = "Alert Settings", desc = "")
+    @Accordion
+    public BroodmotherSpawnAlertConfig spawnAlertConfig = new BroodmotherSpawnAlertConfig();
```

Comment:
remove the suffix "config" from the variable. only the class name should have the name

Reply:
spawnAlert already being used for the config option above it